trikot-viewmodels-declarative-flow/viewmodels-declarative-flow/src/commonMain/kotlin/com/mirego/trikot/viewmodels/declarative/properties/VMDSpanStyleResource.kt

```diff
@@ -0,0 +1,14 @@
+package com.mirego.trikot.viewmodels.declarative.properties
+
+/**
+ * A span style resource within a known set of styles contained in the application. [VMDSpanStyleResource.None]
+ * is used to represent the absence of style. An enum containing all the known span styles usually
+ * conforms to this interface.
+ */
+interface VMDSpanStyleResource {
```

Comment:
Can this directly be VMDStyleResource instead of being directly linked to a span?

Reply:
This is now renamed to `VMDTextStyleResource`